observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
+* Previously, the `oc` `must-gather` plugin for the Network Observability Operator was only working on the `amd64` architecture and failing on all others because the plugin was using `amd64` for the `oc` binary. Now, the Network Observability Operator `oc` `must-gather` plugin collects logs on any architecture platform. (link:https://issues.redhat.com/browse/NETOBSERV-1618[*NETOBSERV-1618*])
+* Previously, when filtering on IP addresses using `not equal to`, the Network Observability Operator would return a request error.
+Now, the IP filtering works in both `equal` and `not equal to` cases for IP addresses and ranges. (link:https://issues.redhat.com/browse/NETOBSERV-1630[*NETOBSERV-1630*])
+* Previously, when a user was not an admin, the error messages were not consistent with the selected tab of the *Network Traffic* view in the web console. Now, the `user not admin` error displays on any tab with improved display.(link:https://issues.redhat.com/browse/NETOBSERV-1621[*NETOBSERV-1621*])
+
+[id="network-observability-operator-1.6.0-known-issues_{context}"]
+=== Known issues
+* When the eBPF agent `PacketDrop` feature is enabled, and sampling is configured to a value greater than `1`, reported dropped bytes and dropped packets ignore the sampling configuration. While this is done on purpose to not miss any drops, a side effect is that the reported proportion of drops versus non-drops becomes biased. For example, at a very high sampling rate, such as `1:1000`, it is likely that almost all the traffic appears to be dropped when observed from the console plugin. (link:https://issues.redhat.com/browse/NETOBSERV-1676[*NETOBSERV-1676*])
+* In the *Manage panels* pop-up window in the *Overview* tab, filtering on *total*, *bar*, *donut*, or *line* does not show any result. (link:https://issues.redhat.com/browse/NETOBSERV-1540[*NETOBSERV-1540*])
+* The SR-IOV secondary interface is not detected if the interface was created first and then the eBPF agent was deployed. It is only detected if the agent was deployed first and then the SR-IOV interface is created. (link:https://issues.redhat.com/browse/NETOBSERV-1697[*NETOBSERV-1697*])
+* When Loki is disabled, the *Topology* view in the OpenShift web console always shows the *Cluster* and *Zone* aggregation options in the slider beside the network topology diagram, even when the related features are not enabled. There is no specific workaround, besides ignoring these slider options. (link:https://issues.redhat.com/browse/NETOBSERV-1705[*NETOBSERV-1705*])
+* When Loki is disabled, and the OpenShift web console first loads, it might display an error: `Request failed with status code 400 Loki is disabled`. As a workaround, you can continue switching content on the *Network Traffic* page, such as clicking between the *Topology* and the *Overview* tabs. The the error should disappear. (link:https://issues.redhat.com/browse/NETOBSERV-1706[*NETOBSERV-1706*])
```

Comment:
```suggestion
* When Loki is disabled, the *Topology* view in the OpenShift web console always shows the *Cluster* and *Zone* aggregation options in the slider beside the network topology diagram, even when the related features are not enabled. There is no specific workaround, besides ignoring these slider options. (link:https://issues.redhat.com/browse/NETOBSERV-1705[*NETOBSERV-1705*])
* When Loki is disabled, and the OpenShift web console first loads, it might display an error: `Request failed with status code 400 Loki is disabled`. As a workaround, you can continue switching content on the *Network Traffic* page, such as clicking between the *Topology* and the *Overview* tabs. The error should disappear. (link:https://issues.redhat.com/browse/NETOBSERV-1706[*NETOBSERV-1706*])
```